libs/types/src/lib.rs

```diff
@@ -230,3 +230,24 @@ pub struct XcmMetadata {
 	/// When `None`, the `default_per_second` will be used instead.
 	pub fee_per_second: Option<Balance>,
 }
+
+pub mod consts {
+	use frame_support::parameter_types;
+
+	use super::*;
```

Comment:
Because this is within a `mod` and we need to import the things we have already imported in the super context.

Reply:
Or is this canonical behaviour in rust?

Reply:
`paremeter_types`, and all of these `#[derive(TypeInfo, Eq, PartialEq, Debug, Clone, Copy )]`. If these imports weren't needed, `cargo fmt` and `clippy` should be upset which is covered by the CI already.

Reply:
Note that we could just import those specifically, but since we already do that in the `super` context, I just import everything from `super` to avoid duplicating imports within the same rust file.

Reply:
@mustermeiszer actually, we can't use the constants here because those constants are actually `parameter_types` and need to be both unwrapped (`Get::get()`) and casted from `u32` to `usize`, which can't happen when defining a fixed-size array. Will leave it as is. The tests ensure these values are the same so we are safe.